.github/workflows/airflow-operator.yml

```diff
@@ -100,7 +100,7 @@ jobs:
           # Manually create folders to ensure perms are correct.
           mkdir -p .kube/internal
           mkdir -p .kube/external
-          go run github.com/magefile/mage@v1.14.0 -v localdev minimal
+          go run github.com/magefile/mage@v1.14.0 -v localdev minimal-legacy
```

Comment:
I'm not familiar with this part of Armada, so I've refrained from moving it to the Pulsar scheduler.